### PR TITLE
First fixes 131: components, $(MAKE) and reports

### PR DESCRIPTION
**Pull request by matentzn**
- Added shortcut make commands for recreating components: `recreate-components` and `recreate-%`
- Generated ROBOT reports now consider the imports by default
- Run internal `make` and $(MAKE) to ensure that parameters are propagated correctly ([issue](https://github.com/INCATools/ontology-development-kit/issues/554))
Mirror goal is only run when mirrors have been downloaded

### CODE REVIEW

**Comment by matentzn** [comment]
@gouttegd sorry I bundled a few things into one PR while I was working on many problems at once.. Would like to merge this to return to a more.. atomic form of PRs.. :) Would you mind taking a look?

**Review by gouttegd — CHANGES_REQUESTED** [review]
Looks good to me, modulo the fact that `js2cli` should be declared in `requirements.txt.lite` _and_ in `requirements.txt`.

File: requirements.txt.lite [hunk 1]
```diff
@@ -11,3 +11,5 @@ pyyaml
 ruamel.yaml
 dosdp
 tsvalid
+j2cli[yaml]
+j2cli
```
[the remark this answers is not in the document]
Comment (matentzn) [reply requirements.txt.lite:15]: I assume you mean `requirements.txt.full`?

[the remark this answers is not in the document]
Comment (gouttegd) [reply requirements.txt.lite:15]: Yes, my bad.

**Comment by matentzn** [comment]
Else is fine, can I merge?

**Review by gouttegd — APPROVED** [review]

**Pull request merged**